script/deploy.s.sol

```diff
@@ -126,7 +126,7 @@ contract DeployerScript is Create2Helper, ArtifactsReader {
         bytes memory bytecode = abi.encodePacked(type(KintoID).creationCode, abi.encode(address(factory)));
         (, address implementation) = _deploy("KintoID", type(KintoID).creationCode, bytecode);
         privateKey > 0 ? vm.broadcast(privateKey) : vm.broadcast();
-        kintoID.upgradeTo(implementation);
+        kintoID.upgradeToAndCall(implementation, bytes(""));
```

Comment:
`upgradeTo` does not exist anymore and has been replaced for `upgradeToAndCall` (more info [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/CHANGELOG.md?plain=1#L112))